java/src/jmri/jmrix/sprog/SprogConstants.java

```diff
@@ -82,11 +72,21 @@
     public final static int SLOT_FREE = 0;
     public final static int SLOT_IN_USE = 1;
 
+    /*
+     * Maximum number of slots for soft command station 
+     * 
+     * More slots allows more throttles to be opened but the refresh rate for
+     * each throttle will reduce.
+     * 
+     */
+    /* Default */
+    public static int MAX_SLOTS = 16;
```

Comment:
Just a thought. No need to fix.
But since `MAX_SLOTS` is a default max and not a hard max, it could be better to name it `DEFAULT_MAX_SLOTS` to show that this setting can be overridden by the user.

Reply:
Thanks for spotting that. MAX_SLOTS should also no longer be public, since overriding it does not work as intended. I will fix these issues for the sake of completeness, if nothing else.